i18n/en/building.md

```diff
@@ -152,6 +152,7 @@ pacman -Su
 * Install dependencies: ```pacman -S make mingw-w64-x86_64-cmake mingw-w64-x86_64-gcc mingw-w64-x86_64-boost mingw-w64-x86_64-openssl```
 * Optional: ```mingw-w64-x86_64-doxygen```  (you'll need [Graphviz](http://graphviz.org/doc/winbuild.html) for doxygen)
 * Note: You'll need  ``` mingw-w64-x86_64-miniupnpc``` if you are behind a restrictive NAT firewall.
+* Note: Ensure ```C:\<path-to>\msys64\mingw64\bin``` (or mingw32) is on your environment path when running MSYS2 bash from PowerShell
```

Comment:
This is not the proper way to start msys2/mingw. Why do you want to do this? How is this kovri-specific?

Reply:
I'm new to msys2/mingw development, this is my mistake.